04_numeronym/SandhyaSMani/04_numeronym_test.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var tests = []struct {
+	in  string
+	out []string
+}{
+	{"numeronym", []string{"n7m"}},
+	{"numeronym test", []string{"n12t"}},
+	{"abc", []string{"abc"}},
+	{"abcd", []string{"a2d"}},
+	{"ab", []string{"ab"}},
```

Comment:
Add unicode words (unicode at beginning and middle), empty string, empty slice to your tests please as well as space at beginning and middle of words.